infra/aws/terraform/modules/kms/kms.tf

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+resource "aws_kms_key" "main" {
+  description             = var.description
+  deletion_window_in_days = var.deletion_window_in_days
+  is_enabled              = true
+  multi_region            = true
```

Comment:
I don't think we need a multi region key. I'm not opposed, just it might not be needed.

Reply:
Removed.